a.core_concepts.md

```diff
@@ -170,7 +200,9 @@ kubectl get po --all-namespaces
 <p>
 
 ```bash
-kubectl run nginx --image=nginx --restart=Never --port=80
+kubectl run nginx2 --image=nginx $do --expose --port 80 
```

Comment:
Why do you create a service for this with --expose?  It seems unneeded based off what the question is asking and should be
```kubectl run nginx2 --image=nginx --port=80 $do```